check.js

```diff
@@ -0,0 +1,39 @@
+let Parser = require('rss-parser');
+let parser = new Parser();
+const { exec} = require('child_process');
+const boxen = require('boxen');
+var cv = "v0.1.17";
```

Comment:
If you are confused just tell me and i can walk you through it.Not editing this one line could cause an issue with starting even when at latest version!!!!!

Reply:
Line 5 is the one i am talking about.